CONTRIBUTION.md

```diff
@@ -0,0 +1,137 @@
+# Contributing to Trace Viewer extension
+
+Thanks for your interest in the [Trace Viewer extension][trace-viewer]! The following is a set of 
+guidelines for contributing to the Trace Viewer extension for Theia applications.
+
+## How to Contribute
+
+⚠️ **Linux only!** Currently, the development environment for this project only works on Linux.
+
+In order to contribute, please first [open an issue][issues] that clearly describes the bug you 
+intend to fix or the feature you would like to add. Make sure you provide a way to reproduce the bug 
+or test the proposed feature. 
+
+Once you have your code ready for review, please  [open a pull request][pull-requests]. Please follow
+the [pull request guidelines][pr-guide]. If you are new to the project, the 
+[new contributors][new-contributors] section provides useful information to get you started. A 
+committer of the Trace Extension will then review your contribution and help to get it merged.
+
+Please note that before your pull request can be accepted, you must electronically sign the 
+[Eclipse Contributor Agreement][eca]. For more information, see the 
+[Eclipse Foundation Project Handbook][commiter-handbook].
+
```

Comment:
Line 19-22 can be removed, but move the 3 chapters for code-of-conduct, dev process and ECA move here.